lib/utils/matrix_sdk_extensions/event_extension.dart

```diff
@@ -169,9 +169,19 @@ extension LocalizedBody on Event {
   bool get isPinned => room.pinnedEventIds.contains(eventId);
 
   Future<bool> unpin() async {
-    if (isPinned) {
-      room.pinnedEventIds.remove(eventId);
-      await room.setPinnedEvents(room.pinnedEventIds);
+    if (!isPinned) {
+      return true;
+    }
+    final pinnedEventIds = room.pinnedEventIds;
+    pinnedEventIds.remove(eventId);
+    String? message;
+    try {
+      message = await room.setPinnedEvents(pinnedEventIds);
+    } catch (e) {
+      Logs().e('$e');
```

Comment:
please also print out the stack trace too

Reply:
don't have stackTrace for this